app/Module/Web/Component/FilterControl/FilterControl.php

```diff
@@ -94,6 +94,8 @@ protected function createComponentForm($name)
         $form->onSuccess[] = [$this, 'setFilter'];
 
         $this->appendAccessibility($form);
+        $this->appendAccessibilityPram($form);
```

Comment:
Moje představa, popsaná ve specifikaci, byla taková, že na frontendu i v API se bude vybírat kategorie přísutpnosti s možnými hodnotami: tělesně postižení, rodiče s dětmi, senioři. A filtr na vlastní přístupnost zůstane pouze jeden (stávající).

To by znamenalo tento commit prakticky zahodit, takže další části komnentovat nebudu (kromě 1 výjimky).

Řešení, které jsi v tomto commitu rozpracoval je taky použitelné. Ale musí být podle mě doplněné o výběr té kategorie a podle té se bude aplikovat pouze jeden filtr na přístupnost. Jinými slovy nechceme ani na webu ani přes API umožnit současně filtrovat dle přístupnosti pro tělesně postižené, seniory i rodiče zároveň.